scripts/wallet.js

```diff
@@ -250,6 +251,31 @@ export class Wallet {
         return !!this.#masterKey;
     }
 
+    /**
+     * Check if the current encrypted keyToBackup can be decrypted with the given password
+     * @param {string} strPassword
+     * @return {Promise<boolean>}
+     */
+    async decryptWallet(strPassword = '') {
+        // Check if there's any encrypted WIF available
+        const database = await Database.getInstance();
+        const { encWif: strEncWIF } = await database.getAccount();
+        if (!strEncWIF || strEncWIF.length < 1) return false;
+
+        // Prompt to decrypt it via password
+        const strDecWIF = await decrypt(strEncWIF, strPassword);
+        if (!strDecWIF || strDecWIF === 'decryption failed!') {
+            if (strDecWIF)
+                return createAlert('warning', ALERTS.INCORRECT_PASSWORD, 6000);
```

Comment:
```suggestion
                return false;
```
`Wallet.decrypt` should work like `Wallet.encrypt` (in #248), it should return a boolean without alerting